flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/KafkaConsumerCallBridge010.java

```diff
@@ -34,6 +38,13 @@
  */
 public class KafkaConsumerCallBridge010 extends KafkaConsumerCallBridge {
 
+	private Date startupDate;
```

Comment:
Passing in the startup date to the API call bridge constructor seems to be very confusing ...